lib/screens/config/v2/cr_departures.ex

```diff
@@ -0,0 +1,45 @@
+defmodule Screens.Config.V2.CRDepartures do
+  @moduledoc false
+
+  alias Screens.V2.WidgetInstance
+
+  @type platform_directions ::
+          %{
+            left: list(pos_integer()),
+            right: list(pos_integer())
+          }
+          | String.t()
+
+  @type t :: %__MODULE__{
+          station: String.t(),
+          destination: String.t(),
+          direction_to_destination: 0 | 1,
+          priority: WidgetInstance.priority(),
+          travel_time_to_destination: String.t(),
+          show_via_headsigns_message: true | false,
+          wayfinding_arrows: platform_directions()
```

Comment:
🤔 I've tested this logic using the 2 sample configs in the PR description with no issue. What do you mean "try adding some sample config to test the logic"?

Reply:
Huh ok! If it works then I guess it's fine. I did not expect keys in the `wayfinding_arrows` sub-object to properly get converted from strings `"left"` `"right"` to atoms `:left` `:right`. Normally `Jason.decode/1` maintains keys from JSON objects as strings, and we have to go through and convert them to atoms.

Reply:
Oh, it's working because none of our backend code actually tries to use access those keys. The map just gets dropped into the return value of `serialize/1` unchanged, and `Jason.encode/1` handles the string keys without a problem when converting back to a JSON string to be sent back in the response.

Reply:
I see! I'll roll with this as acceptable, then, unless you think otherwise